R/internals.R

```diff
@@ -245,6 +245,13 @@
   return(clean_val)
 }
 
+.get_understat_json <- function(page_url) {
+  tryCatch(
+    httr::GET(page_url, httr::set_cookies(.cookies = c("beget" = "begetok"))) %>% httr::content(),
```

Comment:
this is the major change. i think what happens is the page is loaded twice now. the first time, the page loads on the server-side as it used to + adds the cookie requirement, then loads a second time. if the cookie isn't provided by the user on the second request, then the page is not loaded. i'm guessing they implemented this to (try to) prevent scraping

Reply:
Such a good find! Thanks heaps